app/(storyblok)/loading.tsx

```diff
@@ -9,18 +9,15 @@ const Loading = () => {
     <div className='bg-black'>
       <Masthead />
       <main>
-        <Container width='site' className='rs-my-8'>
+        <Container width='site' className='rs-my-10'>
           <FlexBox gap direction='row' wrap='wrap'>
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='w-full'/>
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={200} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[40%] flex-grow' />
+            <Skeleton height={500} highlightColor="rgba(255,255,255,0.2)" baseColor="rgba(0,0,0,0.6)" containerClassName='w-full'/>
+            <Skeleton height={300} highlightColor="rgba(255,255,255,0.2)" baseColor="rgba(0,0,0,0.6)" containerClassName='w-full'/>
```

Comment:
@sherakama 😂

Reply:
Uh, yeah, I'm totally messing with those in https://github.com/SU-SWS/ood-giving-campaign/pull/200.

I was having issues with https://www.npmjs.com/package/react-loading-skeleton and the way they exported the package so I switched over to https://github.com/danilowoz/react-content-loader

I think the options in there look nice.

![Screenshot 2024-01-24 at 09 14 37](https://github.com/SU-SWS/ood-giving-campaign/assets/550602/8ac15095-d7ed-412f-a5e3-dd9b1fb3df05)

Reply:
Keep going on your PR and I'll adjust later. The PR I have is for the future stuff so we don't have to change right away.

Reply:
Oh Cool - thanks for the heads up 😄